packages/lambdas/src/shared/file-info.ts

```diff
@@ -0,0 +1,25 @@
+import { makeS3Client } from "@metriport/core/external/aws/s3";
+import { getEnvOrFail } from "./env";
+
+const region = getEnvOrFail("AWS_REGION");
+const s3client = makeS3Client(region);
+
+export async function getFileInfoFromS3(
+  key: string,
+  bucket: string
+): Promise<
+  | { exists: true; size: number; contentType: string }
+  | { exists: false; size?: never; contentType?: never }
+> {
+  try {
+    const head = await s3client
+      .headObject({
+        Bucket: bucket,
+        Key: key,
+      })
+      .promise();
+    return { exists: true, size: head.ContentLength ?? 0, contentType: head.ContentType ?? "" };
+  } catch (err) {
+    return { exists: false };
+  }
+}
```

Comment:
I think we have this on the API, right? Let's move those to `packages/core` and call that from both places, please

Reply:
Just realized this is actually on the doc downloader lambda, which I'm refactoring. So let's leave this here, just add a TODO to use core's instead? I'll make sure to have this as a reusable function there.

Reply:
Oh nice! Thanks 👍

Reply:
I cound't find it on core, but I'll make it available. It's currently on the api and on here: `metriport/packages/lambdas/src/shared/s3/S3Utils`

Reply:
Oh, apologies, I saw it there and forgot I introduced it [here](https://github.com/metriport/metriport/pull/1013).